src/content/topics/integrations/more/compass.mdx

```diff
@@ -0,0 +1,144 @@
+---
+path: /integrations/compass
+title: Compass
+description: This topic explains visualize LaunchDarkly feature flag changes in Atlassian Compass.
+published: true
+---
+
+<Callout intent="primary">
+<CalloutTitle>The Compass integration is a Pro and Enterprise feature</CalloutTitle>
+<CalloutDescription>
+
+The Compass integration is available to customers on a Pro or Enterprise plan. To learn more, [read about our pricing](https://launchdarkly.com/pricing/). To upgrade your plan, [contact Sales](https://launchdarkly.com/contact-sales/).
+
+</CalloutDescription>
+</Callout>
+
+## Overview
+
+This topic explains how to use the Compass integration for LaunchDarkly.
+
+<Callout intent="info">
+<CalloutTitle>The Compass integration is managed by Atlassian</CalloutTitle>
+<CalloutDescription>
+
+The Compass integration was built by and is managed by Atlassian. For questions about the integration or for technical support, [contact Atlassian](https://community.atlassian.com/t5/Compass-Alpha/gh-p/compass-alpha).
+
+</CalloutDescription>
+</Callout>
+
+The Compass integration for LaunchDarkly displays a timeline of feature flag changes related to a Compass component, as well as a complete list of feature flags and their current state. This helps you visualize how recent changes to feature flags may be impacting component health.
+
+## Prerequisites
+
+To connect Compass to LaunchDarkly, you must meet the following prerequisites:
+
+1. Be a Compass administrator on your Atlassian site
+2. Generate a LaunchDarkly [API access token](/home/account-security/api-access-tokens) for use with Compass.
+* Create a token with the **Reader** role.
+* If you are on an Enterprise plan, check the **This is a service token** option when creating your acesss token.
+* If you are on a Pro plan, leave this option unchecked.
+* Be sure to copy the new token to your clipboard.
+
+## Setting up the Compass integration
+
+The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.
+
+1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
```

Comment:
```suggestion
The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.

Here's how to set up your Compass integration:

1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
```

We typically introduce each procedure with a short stem sentence to be explicit about exactly what the procedure is doing.